main.py

```diff
@@ -117,36 +118,24 @@ def run(filename, rule_type=RuleType.ALL, with_console_output=False, execution_m
                 feature_file = item['location'].split(':')[0]
                 shas = get_commits(cwd, feature_file)
                 version = len(shas)
-                check_disabled = 'disabled' in item['tags']
-                if check_disabled:
-                    yield f"{feature_name}.v{version}", f"{remote}/blob/{shas[0]}/{feature_file}", "Rule disabled", ("Rule disabled", "This rule has been disabled from checking"), "Rule disabled", "Rule disabled"
-
+                rule_is_disabled = 'disabled' in item['tags']
+                yield { # add feature information, mainly to check disabled feature
+                    'feature_name': f"{feature_name}.v{version}",
+                    'remote' : f"{remote}/blob/{shas[0]}/{feature_file}",
```

Comment:
I don't think we depend on this anymore.

Reply:
The `'remote'` I mean

Reply:
No, [deleted it ](https://github.com/buildingSMART/ifc-gherkin-rules/commit/101ec619eb8815e7f074479aaf4199ab1a8f3800)
The feature version is also unused at the moment. I left it in since I have a feeling we'll use this versioning more in the future.